<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMainWindow</class>
 <widget class="QMainWindow" name="QWMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>573</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>多窗口应用程序</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>811</width>
      <height>461</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="action_N_Ctrl_N"/>
    <addaction name="action_W_Ctrl_Shfit_N"/>
    <addaction name="action_O_Ctrl_O"/>
    <addaction name="action_S_Ctrl_S"/>
    <addaction name="action_A_Ctrl_Shift_S_U"/>
    <addaction name="separator"/>
    <addaction name="action_U_2"/>
    <addaction name="action_P_Ctrl_P_2"/>
    <addaction name="action_X_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑(E)</string>
    </property>
    <addaction name="action_U_Ctrl_Z"/>
    <addaction name="separator"/>
    <addaction name="action_T_Ctrl_X"/>
    <addaction name="action_C_Ctrl_C"/>
    <addaction name="action_P_Ctrl_V"/>
    <addaction name="action_L_Del"/>
    <addaction name="separator"/>
    <addaction name="action_F_Ctrl_F"/>
    <addaction name="action_N_F3"/>
    <addaction name="action_N_Shfit_F3"/>
    <addaction name="action_R_Ctrl_H"/>
    <addaction name="action_G_Ctrl_G"/>
    <addaction name="separator"/>
    <addaction name="action_A_Ctrl_A"/>
    <addaction name="action_D_F5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式(O)</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actWidgetInsite"/>
   <addaction name="actionNewNoteBook"/>
   <addaction name="separator"/>
   <addaction name="actWidget"/>
   <addaction name="actWindowInsite"/>
   <addaction name="actWindow"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actWindowInsite">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/2.ico</normaloff>:/images/images/2.ico</iconset>
   </property>
   <property name="text">
    <string>嵌入式MainWindow</string>
   </property>
   <property name="toolTip">
    <string>嵌入式MainWindow</string>
   </property>
  </action>
  <action name="actWidgetInsite">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/100.bmp</normaloff>:/images/images/100.bmp</iconset>
   </property>
   <property name="text">
    <string>嵌入式Widget</string>
   </property>
   <property name="toolTip">
    <string>Widget嵌入式窗体</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actDoc_Open">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/122.bmp</normaloff>:/images/images/122.bmp</iconset>
   </property>
   <property name="text">
    <string>打开文档</string>
   </property>
   <property name="toolTip">
    <string>打开文档</string>
   </property>
  </action>
  <action name="actWindow">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/3.ico</normaloff>:/images/images/3.ico</iconset>
   </property>
   <property name="text">
    <string>独立MainWindow窗口</string>
   </property>
   <property name="toolTip">
    <string>独立MainWindow窗口</string>
   </property>
  </action>
  <action name="actWidget">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/804.bmp</normaloff>:/images/images/804.bmp</iconset>
   </property>
   <property name="text">
    <string>独立Widget窗口</string>
   </property>
   <property name="toolTip">
    <string>新建Widget独立窗口</string>
   </property>
  </action>
  <action name="actionNewNoteBook">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/516.bmp</normaloff>:/images/images/516.bmp</iconset>
   </property>
   <property name="text">
    <string>新建笔记本</string>
   </property>
   <property name="toolTip">
    <string>创建新的笔记本</string>
   </property>
  </action>
  <action name="action_N_Ctrl_N">
   <property name="text">
    <string>新建(N)                   Ctrl+N</string>
   </property>
  </action>
  <action name="action_W_Ctrl_Shfit_N">
   <property name="text">
    <string>新窗口(W)       Ctrl+Shfit+N</string>
   </property>
  </action>
  <action name="action_O_Ctrl_O">
   <property name="text">
    <string>打开(O)...                 Ctrl+O</string>
   </property>
  </action>
  <action name="action_S_Ctrl_S">
   <property name="text">
    <string>保存(S)                     Ctrl+S</string>
   </property>
  </action>
  <action name="action_A_Ctrl_Shift_S_U">
   <property name="text">
    <string>另存为(A)          Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_U">
   <property name="text">
    <string>
页面设置(U)...</string>
   </property>
  </action>
  <action name="action_P_Ctrl_P">
   <property name="text">
    <string>打印(P)...            Ctrl+P</string>
   </property>
  </action>
  <action name="action_X">
   <property name="text">
    <string>退出(X)</string>
   </property>
  </action>
  <action name="action_U_2">
   <property name="text">
    <string>页面设置(U)...              </string>
   </property>
  </action>
  <action name="action_P_Ctrl_P_2">
   <property name="text">
    <string>打印(P)...                    Ctrl+P</string>
   </property>
  </action>
  <action name="action_X_2">
   <property name="text">
    <string>退出(X)                     </string>
   </property>
  </action>
  <action name="action_U_Ctrl_Z">
   <property name="text">
    <string>撤销(U)                   Ctrl+Z</string>
   </property>
  </action>
  <action name="action_T_Ctrl_X">
   <property name="text">
    <string>剪切(T)                   Ctrl+X</string>
   </property>
  </action>
  <action name="action_C_Ctrl_C">
   <property name="text">
    <string>复制(C)                   Ctrl+C</string>
   </property>
  </action>
  <action name="action_P_Ctrl_V">
   <property name="text">
    <string>粘贴(P)                   Ctrl+V</string>
   </property>
  </action>
  <action name="action_L_Del">
   <property name="text">
    <string>删除(L)                        Del</string>
   </property>
  </action>
  <action name="action_F_Ctrl_F">
   <property name="text">
    <string>查找(F)                   Ctrl+F</string>
   </property>
  </action>
  <action name="action_N_F3">
   <property name="text">
    <string>查找下一个(N)               F3</string>
   </property>
  </action>
  <action name="action_N_Shfit_F3">
   <property name="text">
    <string>查找上一个(N)       Shfit+F3</string>
   </property>
  </action>
  <action name="action_R_Ctrl_H">
   <property name="text">
    <string>替换(R)...                Ctrl+H</string>
   </property>
  </action>
  <action name="action_G_Ctrl_G">
   <property name="text">
    <string>转到(G)...                Ctrl+G</string>
   </property>
  </action>
  <action name="action_A_Ctrl_A">
   <property name="text">
    <string>全选(A)                   Ctrl+A</string>
   </property>
  </action>
  <action name="action_D_F5">
   <property name="text">
    <string>时间/日期(D)                  F5</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>QWMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
